crates/transaction-emitter-lib/src/emitter/account_minter.rs

```diff
@@ -384,15 +409,35 @@ where
                     )
                 })
                 .collect();
-            execute_and_wait_transactions(
+            let result = execute_and_wait_transactions(
                 &client,
                 &mut source_account,
                 creation_requests,
                 failed_requests,
             )
             .await
-            .with_context(|| format!("Account {} couldn't mint", source_account.address()))?;
-            batch
+            .with_context(|| format!("Account {} couldn't mint", source_account.address()));
+
+            if let Err(err) = result {
+                warn!("Creating seed accounts batch failed {:?}, retrying", err);
+                loop {
+                    let result = client.get_account(source_account.address()).await;
```

Comment:
Same as above and also, maybe try to see if it's feasible to re-use the code above.